connectors/webhook/src/main/java/io/camunda/connector/inbound/HttpWebhookExecutable.java

```diff
@@ -120,5 +131,14 @@ public void activate(InboundConnectorContext context) throws Exception {
     }
     props = new WebhookConnectorProperties(context.getProperties());
     context.replaceSecrets(props);
+
+    // jwk url must be specified in the element template for this to work
+    if (props.getJwkUrl() != null) {
```

Comment:
We should only do this if `WebhookConnectorProperties.AuthorizationType.JWT.equals(props.getAuthorizationType()` right?